### DISCUSSION TITLE
Copy and Pasting HTML content

### DISCUSSION THREAD

**Discussion by maelpetit** [ueberdosis/tiptap#1020] [General Questions]
Hi,

I'm trying to paste HTML content to my tiptap editor but a `<p>` is added to every line ruining the look of the pasted content.

Is there any way to prevent this from happening ? I can't use `clipboardTextParser` because it is not called for HTML content and `transformPastedHTML` is too early. I want a `clipboardHTMLParser` but there is none.

Thanks in advance fo any insight :smile:

**Comment by varshiniNaN** [comment]
> Hi,
> 
> I'm trying to paste HTML content to my tiptap editor but a `<p>` is added to every line ruining the look of the pasted content.
> 
> Is there any way to prevent this from happening ? I can't use `clipboardTextParser` because it is not called for HTML content and `transformPastedHTML` is too early. I want a `clipboardHTMLParser` but there is none.
> 
> Thanks in advance fo any insight 😄

@maelpetit  I'm also facing the same issue. Did you find any solution?
